docs/metrics/pearson-correlation.md

```diff
@@ -0,0 +1,54 @@
+# Pearson Correlation Coefficient
```

Comment:
Add a `description` to the front matter please — see other metrics guides or #609 for examples.